### PR TITLE
Update the term "Population" to "Genetic Ancestry Group", and "popmax" to "grpmax" on user facing portions of the browser

### LINKED ISSUE #1051: Update population/ancestry terms
Related to https://github.com/broadinstitute/gnomad_production/issues/707: we received feedback from external users that they prefer to use the term "genetic ancestry" rather than "population" or "ancestry" for clarity

### PR DESCRIPTION
**Pull request by rileyhgrant**
Resolves #1051 

Updates 

- the term "Population" to "Genetic ancestry group", 
- the term "popmax" to "grpmax" 

throughout the user facing text of browser.

---

Changing the term usage in the API, and possibly local variable names if we want, is a task for later.

### CODE REVIEW

File: browser/src/CopyNumberVariantPage/__snapshots__/CopyNumberVariantPage.spec.tsx.snap [hunk 1]
```diff
@@ -779,7 +779,7 @@ exports[`CopyNumberVariantPage with dataset gnomad_cnv_r4 with variant of type D
   </div>
   <section>
     <h2>
-      Population Frequencies
+      Genetic ancestry group Frequencies
```
Comment (mattsolo1) [inline browser/src/CopyNumberVariantPage/__snapshots__/CopyNumberVariantPage.spec.tsx.snap:782]: All section titles (like on variant pages) should be in title case (Genetic Ancestry Group Frequencies)

File: browser/src/CopyNumberVariantPage/__snapshots__/CopyNumberVariantPage.spec.tsx.snap [hunk 2]
```diff
@@ -795,7 +795,7 @@ exports[`CopyNumberVariantPage with dataset gnomad_cnv_r4 with variant of type D
               onClick={[Function]}
               type="button"
             >
-              Population
+              Genetic ancestry group
```
Comment (mattsolo1) [inline browser/src/CopyNumberVariantPage/__snapshots__/CopyNumberVariantPage.spec.tsx.snap:798]: Can probably keep lower case

**Review by mattsolo1 — CHANGES_REQUESTED** [review]
You could also change all user-facing text (not the API query/code yet) of Popmax -> Grpmax as part of this PR.

**Comment by rileyhgrant** [comment]
> You could also change all user-facing text (not the API query/code yet) of Popmax -> Grpmax as part of this PR.

Oop yeah, missed that, good call will do.

File: browser/src/CopyNumberVariantPage/__snapshots__/CopyNumberVariantPage.spec.tsx.snap (hunk 2 above)
Comment (rileyhgrant) [reply browser/src/CopyNumberVariantPage/__snapshots__/CopyNumberVariantPage.spec.tsx.snap:798]: Seems like the other column headers are title case so going with that for now, happy to do sentence case instead if you prefer.

Thanks for catching the case consistency. 🙏

**Comment by mattsolo1** [comment]
Actually I wonder if it should be Groupmax fully spelled out? @ch-kr?

**Review by mattsolo1 — APPROVED** [review]

**Pull request merged**